bin/zkCli.cmd

```diff
@@ -17,7 +17,7 @@ REM limitations under the License.
 setlocal
 call "%~dp0zkEnv.cmd"
 
-ZOO_LOG_FILE=zookeeper-%USERNAME%-cli-%COMPUTERNAME%.log
+set ZOO_LOG_FILE=zookeeper-%USERNAME%-cli-%COMPUTERNAME%.log
 
```

Comment:
- this fix seems reasonable. I will test this in my windows OS at night.
- BTW. @jorgheymans  Could you plz create a [jira](https://issues.apache.org/jira/projects/ZOOKEEPER/issues/ZOOKEEPER-3211?filter=allopenissues) binding this PR?

Reply:
I reproduce this issue in my windows, this fix is good.

Reply:
@maoling sorry for the delay, here is the jira ticket https://issues.apache.org/jira/browse/ZOOKEEPER-3453

Reply:
change this PR title to "`ZOOKEEPER-3453:zkcli.cmd: ZOO_LOG_FILE is missing set`", making it bind to the JIRA.